build.gradle

```diff
@@ -37,4 +37,12 @@ subprojects {
     tasks.withType(Javadoc) {
       options.addStringOption('Xdoclint:none', '-quiet')
     }
+
```

Comment:
@kilink , try pulling this change into your branch and run with 
```
./gradlew  -DtestJdk=17 clean test
```
to run the tests under 17.
As of this change, I still have a bunch of failures, but they seem to mostly be related to dep mismatches with Mockito :-(